core-backend/sandbox/src/env.rs

```diff
@@ -384,7 +394,7 @@ where
             unreachable!("We cannot use sandbox backend in std environment currently");
         };
 
-        match pre_execution_handler(&mut runtime.memory, stack_end, globals_config) {
+        match prepare_memory(&mut runtime.memory, stack_end, globals_config) {
```

Comment:
What if we add some additional functionality to this function?

Reply:
Currently it's a handler, which returns `PrepareMemoryError`, so renaming that seems fine. If complex functionality is introduced, not only name, but the closure signature will change.

Reply:
agreed to resolve